torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12736,7 +12736,8 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                          dtypes=(torch.int, torch.int8)),
             # pytorch computes (0+nanj), numpy computes (-5e-18-1j) for input (-501.-1.0000e+20j)
             DecorateInfo(unittest.expectedFailure, 'TestUnaryUfuncs',
-                         "test_reference_numerics_large", dtypes=(torch.complex64,)),),
+                         "test_reference_numerics_large", dtypes=(torch.complex64,), device_type='cpu',
+                         active_if=not IS_MACOS),),
```

Comment:
I think that the `active_if` is not necessary, as in mac `device_type == "mps"` I believe. In any case, no need to change it for this PR. You can try removing it in the next PR if you feel like it, otherwise it's alright as well.